tutorials/configure-smtp-relay-tem/index.mdx

```diff
@@ -0,0 +1,146 @@
+9. Use the following command to launch Postmap:
+    ```bash
+    postmap /etc/postfix/sasl_passwd
+    ```
+    <Message type="tip">
+     Depending on your configuration, you might need to use `sudo postmap /etc/postfix/sasl_passwd` to be able to launch Postmap.
+    </Message>
+10. Relaunch Postfix using the following command:
+    ```bash
+    sudo postfix start
+    ```
+11. Use the following command to send an email. Make sure that you replace `<your-domain>` with your own domain and `recipient@recipient.com` with your recipient's email address.
+    ```bash
+    date | mail -s "Test from postfix" -a "From:<your-domain>" recipient@recipient.com
+    ```
+
+    <Message type="important">
+    Keep in mind that the **subject**, **body** and **HTML** of your emails must have **at least 10 characters**.
+    </Message>
+12. Optionally, use `mailq` to see if your mail has been sent. An empty mail queue means that your email has been sent.
+
+## Configuring Postfix with a TLS port
+
+Some providers do not let you access port 587. To solve this issue, Scaleway offers non-standard ports that are unlikely to be blocked: `2587` (SMTP) and `2465` (TLS). A workaround is to use stunnel to be able to use Postfix.
+
+1. Open a terminal and access the `smtp-wrapper.conf` file using the following command:
+    ```bash
+    nano /etc/stunnel/smtp-wrapper.conf
+    ```
+2. Copy the following configuration and paste it into the `smtp-wrapper.conf` file:
+    ```bash
+    [smtp-tls-wrapper]
+    accept = 2465
+    client = yes
+    connect = smtp.tem.scw.cloud:2465
+    ```
+3. Type `CTRL O` and press `Enter` to save, then type `CTRL X` to exit the file.
+4. Open a new terminal and access your `sasl_passwd` file within Postfix using the following command:
+    ```bash
+    nano /etc/postfix/sasl_passwd
+    ```
+5. Copy the following configuration and paste it in your file:
+    ```bash
+    [localhost]:2465 <scw_access>:<scw_password>
+    ```
+6. Type `CTRL O` and press `Enter` to save, then type `CTRL X` to exit the file.
+7. Access your `main.cf` file using the following command:
+    ```bash
+    cd .. && nano main.cf
+    ```
+8. Replace `relayhost = [smtp.tem.scw.cloud]:587` with `relayhost = [localhost]:2465`.
+9. Type `CTRL O` and press `Enter` to save, then type `CTRL X` to exit the file.
+10. Use the following command to launch Postmap:
+    ```bash
+    postmap /etc/postfix/sasl_passwd
+    ```
+    <Message type="tip">
+     Depending on your configuration, you might need to use `sudo postmap /etc/postfix/sasl_passwd` to be able to launch Postmap.
+    </Message>
+11. Relaunch Postfix using the following command:
+    ```bash
+    sudo postfix start
+    ```
+12. Use the following command to send an email. Make sure that you replace `<your-domain>` with your own domain and `recipient@recipient.com` with your recipient's email address.
+    ```bash
+    date | mail -s "Test from postfix" -a "From:<your-domain>" recipient@recipient.com
```

Comment:
From needs to be a email address - otherwise the mail get bounced
```suggestion
    date | mail -s "Test from postfix" -a "From:<you@your-domain>" recipient@recipient.com
```